src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,33 @@
+.stars {
+  display: flex;
+  align-content: center;
```

Comment:
The 'align-content' property doesn't have any effect in a single-line flex container. If you intended to center the items on the cross-axis, 'align-items' is sufficient. You should remove 'align-content' to keep the code clean and avoid confusion.